front/src/components/myCar/EngineCard.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from '@emotion/styled';
+import { Flex } from '../common/Flex';
+import { theme } from '../../styles/theme';
+import { Text } from '../common/Text';
+import { engineOptionInterface } from '../../pages/MyCar/Engine';
+
+const EngineCard = ({
+  engine,
+  isSelected,
+}: {
+  engine: engineOptionInterface;
+  isSelected: boolean;
+}) => {
+  return (
+    <EngineBox
+      direction="column"
+      padding={isSelected ? '0px 20.8px' : '2px 22.8px'}
+      height={engine.height}
+      isSelected={isSelected === true}
+    >
+      <Flex justify="space-between">
+        <Text typo="Heading3_Bold">{engine.engineType}</Text>
+        <Text typo="Heading4_Bold">+{engine.price}원</Text>
```

Comment:
.toLocaleString('ko-KR') 를 사용하면 , 를 자동으로 찍어줘요!!
```suggestion
        <Text typo="Heading4_Bold">+{engine.price.toLocaleString('ko-KR')}원</Text>
```

Reply:
아 맞다 그 부분을 깜빡했네요
좋은 팁 감사합니다